stories/Popover/Popover.stories.jsx

```diff
@@ -0,0 +1,201 @@
+import Popover from '../../components/Popover';
+import { Row, Col } from '../../components';
+
+const Template = args => {
+  const { children: Component, ...rest } = args;
+  return (
+    <>
+      <Row style={{ paddingTop: '72px', paddingLeft: '414px' }}>
+        <Col>
+          <Popover {...rest}>{Component}</Popover>
+        </Col>
+      </Row>
+    </>
+  );
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  children: 'childrenText...',
+  trigger: 'triggerText...',
+};
+
+export const NeutralTop = Template.bind({});
+NeutralTop.args = {
+  direction: 'top',
+  placement: 'top',
+  skin: 'neutral',
+  children: 'childrenText...',
+  trigger: 'triggerText...',
+};
+
+export const NeutralBottom = Template.bind({});
+NeutralBottom.args = {
+  direction: 'bottom',
+  placement: 'bottom',
+  skin: 'neutral',
+  children: 'childrenText...',
+  trigger: 'triggerText...',
+};
+
+export const NeutralLeft = Template.bind({});
+NeutralLeft.args = {
+  direction: 'left',
+  placement: 'left',
+  skin: 'neutral',
+  children: 'childrenText...',
+  trigger: 'triggerText...',
+};
+
+export const NeutralRight = Template.bind({});
+NeutralRight.args = {
+  direction: 'right',
+  placement: 'right',
+  skin: 'neutral',
+  children: 'childrenText...',
+  trigger: 'triggerText...',
+};
+
+export const PrimaryTop = Template.bind({});
+PrimaryTop.args = {
+  direction: 'top',
+  placement: 'top',
+  skin: 'primary',
```

Comment:
In the same way that "placement" happens here with the "skin". If there is already an example of using the prop skin, you don't need other examples for all possible "skin" values